### PR TITLE
Modified datatable query to fetch desired Data

### PR DESCRIPTION
**Pull request by nikhilporwal7**
Added UI changes for viewable report of what will be downloaded

@sarafpratik34

### CODE REVIEW

File: EPFODataLoader_WebForm/Default.aspx.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ protected void Page_Load(object sender, EventArgs e)
 
         protected void GreetBotton_Click(object sender, EventArgs e)
         {
-            HelloWorldLabel.Text = $"Hello {TextInput.Text}" ;
+            HelloWorldLabel.Text = $"Hello {TextInput.Text}";
             TextInput.Text = string.Empty;
 
             if (FileUpload1.HasFile)
```
Comment (sarafpratik34) [inline EPFODataLoader_WebForm/Default.aspx.cs:34]: Rid unwanted use of variable ext and try to use string interpolation for conct

File: EPFODataLoader_WebForm/Default.aspx [hunk 2]
```diff
@@ -2,16 +2,28 @@
...
 
-    <div class="jumbotron">
-        <h1>ASP.NET</h1>
-        <p class="lead">ASP.NET is a free web framework for building great Web sites and Web applications using HTML, CSS, and JavaScript.</p>
-        <p><a href="http://www.asp.net" class="btn btn-primary btn-lg">Learn more &raquo;</a></p>
+    <div class="jumbotron" style="overflow: scroll">
+        <h1>Display Uploaded data</h1>
         <asp:Label runat="server" ID="HelloWorldLabel"></asp:Label>
         <asp:TextBox ID="TextInput" runat="server"></asp:TextBox>
-        <asp:Button ID="GreetBotton" runat="server" OnClick="GreetBotton_Click" Text="Say Hello" class="btn btn-primary btn-lg" />
```
Comment (sarafpratik34) [inline EPFODataLoader_WebForm/Default.aspx:11]: Try to use camel case everywhere

File: EPFODataLoader_WebForm/Default.aspx.cs [hunk 3]
```diff
@@ -68,7 +70,7 @@ protected void DownloadTxt(object sender, EventArgs e)
 
                 //DownloadTxt(fileName);
                 result = ExcelUtilities.DumpDBFToTxt(dt).ToString();
-                string txtFilePath = fileName.Replace("DBF", "txt");
```
Comment (sarafpratik34) [inline EPFODataLoader_WebForm/Default.aspx.cs:71]: You can merge 71 and 72 in one

File: EPFODataLoader_WebForm/Default.aspx.cs [hunk 4]
```diff
@@ -45,7 +45,9 @@ protected void GreetBotton_Click(object sender, EventArgs e)
                 gvExcelFile.DataSource = dt;
                 gvExcelFile.DataBind();
 
-                Label1.Text = $"File Uploaded: {FileUpload1.FileName}" ;
+                Label1.Text = $"File Uploaded: {FileUpload1.FileName}";
+            }
+        }
 
         /// <summary>
         /// Help link: //https://stackoverflow.com/a/25718674
```
Comment (sarafpratik34) [inline EPFODataLoader_WebForm/Default.aspx.cs:53]: Server Map path keep in some variable and use that variable everywhere

File: EPFODataLoader_WebForm/Default.aspx.cs (hunk 3 above)
Comment (nikhilporwal7) [reply EPFODataLoader_WebForm/Default.aspx.cs:71]: 71 and 72 can't be merged. There is no connection between those lines

File: EPFODataLoader_WebForm/Default.aspx.cs (hunk 1 above)
Comment (nikhilporwal7) [reply EPFODataLoader_WebForm/Default.aspx.cs:34]: Plan to implement StyleCop for the same.
Intentionally  #left
Created issue in backlog #8

File: EPFODataLoader_WebForm/Default.aspx.cs (hunk 4 above)
Comment (nikhilporwal7) [reply EPFODataLoader_WebForm/Default.aspx.cs:54]: done

**Pull request merged**